spec/cc/engine/analyzers/javascript/main_spec.rb

```diff
@@ -21,7 +21,7 @@
 
       expect(json["type"]).to eq("issue")
       expect(json["check_name"]).to eq("Identical code")
-      expect(json["description"]).to eq("Identical code found in 2 other locations (mass = 11)")
+      expect(json["description"]).to eq("Identical blocks of code found in 3 locations. Consider refactoring.")
```

Comment:
I like the change to mention total blocks of code, though it seems like a product question. 

Are we going to expose the mass elsewhere?